src/test/java/org/salesforce/steps/CreateCampaignSteps.java

```diff
@@ -0,0 +1,66 @@
+package org.salesforce.steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import salesforce.ui.PageTransporter;
+import salesforce.ui.pages.LoginPage;
+import salesforce.ui.pages.lightning.campaign.CampaignCreatedPage;
+import salesforce.ui.pages.lightning.campaign.CampaignPage;
+import salesforce.ui.pages.lightning.campaign.CreateCampaignPage;
+
+public class CreateCampaignSteps {
+
+    private CampaignPage campaignPage;
+    private CampaignCreatedPage campaignCreatedPage;
+    protected LoginPage loginPage;
+    private PageTransporter pageTransporter;
+    String campaignName = "New Campaign Created";
+
+    public CreateCampaignSteps(PageTransporter pageTransporter) {
+        this.pageTransporter = pageTransporter;
+    }
+
+    @Given("I login to salesforce site as an developer user")
+    public void iLoginToSalesforceSiteAsAnDeveloperUser() {
+    }
+
+    @And("I navigate to {string} page")
```

Comment:
I think the @Ands won't work in a Gradle task, set them just as @given @When or @Then, no need to change them in the cucumber file, you can still call @given @When and @Then as @And in the feature file.

Reply:
ok It was solved